R/subject_industry.R

```diff
@@ -1,7 +1,962 @@
+        uniqueregions[3], "</b> was the most common industry for those living in ", textprod(data3), " and <b>",
+        uniqueregions[4], "</b> was the most common industry for those living in ", textprod(data4), "."
+      )
+    } else if (length(uniqueregions) == 5) {
+      data1 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[1])
+      data2 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[2])
+      data3 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[3])
+      data4 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[4])
+      data5 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[5])
+
+      regiontext <- paste(
+        "<b>",
+        uniqueregions[1], "</b> was the most common industry for those currently living in ", textprod(data1), ",
+                      <b>", uniqueregions[2], "</b> was the most common industry for those living in ", textprod(data2), ", <b>",
+        uniqueregions[3], "</b> was the most common industry for those living in ", textprod(data3), ", <b>",
+        uniqueregions[4], "</b> was the most common industry for those living in ", textprod(data4), " and <b>",
+        uniqueregions[5], "</b> was the most common industry for those living in ", textprod(data5), "."
+      )
+    } else if (length(uniqueregions) == 6) {
+      data1 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[1])
+      data2 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[2])
+      data3 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[3])
+      data4 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[4])
+      data5 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[5])
+      data6 <- regionfirstdata %>%
+        filter(regionfirstdata == uniqueregions[6])
+
+      regiontext <- paste(
+        "<b>",
+        uniqueregions[1], "</b> was the most common industry for those currently living in ", textprod(data1), ",
+                      <b>", uniqueregions[2], "</b> was the most common industry for those living in ", textprod(data2), ", <b>",
+        uniqueregions[3], "</b> was the most common industry for those living in ", textprod(data3), ", <b>",
+        uniqueregions[4], "</b> was the most common industry for those living in ", textprod(data4), ", <b>",
+        uniqueregions[5], "</b> was the most common industry for those living in ", textprod(data5), " and <b>",
+        uniqueregions[6], "</b> was the most common industry for those living in ", textprod(data6), "."
+      )
+    } else {
+      # Just going to recreate the above with the collapse flag. Leaving it in
+      # the else area for now, but should cover all of the above if statements in one go.
+      print("### Using new code for region text...")
```

Comment:
Another print()?

Reply:
Yep, that ones definitely me, checking I knew for definite how the if statements were being negotiated. Will delete that too!